utils/gnomad_functions.py

```diff
@@ -372,7 +372,7 @@ def get_rf_runs(data_type: str) -> Dict:
     :rtype: dict
     """
     
-    from gnomad_hail.resources.variant_qc import rf_run_hash_path
+    from gnomad_qc.resources.variant_qc import rf_run_hash_path
```

Comment:
Can you add a `#FIXME` here, as code should be adapted to have no reference to `gnomad_qc`